packages/cxl-ui/scss/cxl-credential.scss

```diff
@@ -0,0 +1,132 @@
+:host {
+  display: flex;
+  margin: auto;
+  text-align: start;
+  color: initial;
+}
+
+.container {
+  position: relative;
+  z-index: 0;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 1464px;
+  height: 1120px;
+  padding: var(--lumo-space-xl);
+  overflow: hidden;
+  font-family: var(--lumo-font-family);
+  font-size: var(--lumo-font-size-m);
+  font-weight: 500;
+  color: var(--lumo-primary-contrast-color);
+  text-transform: uppercase;
+  background-color: #d61f2c;
+
+  &[invisible] {
+    visibility: hidden;
+  }
+
+  * {
+    z-index: 2;
+  }
+
+  #circles {
+    position: absolute;
+    top: 0;
+    left: 520px;
+    z-index: 1;
+    width: 100%;
+    height: 100%;
+
+    svg {
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      margin: auto;
+    }
+  }
+
+  #cxl-logo {
+    width: 128px;
+    height: 54px;
+
+    --lumo-primary-color: #fff;
+  }
+
+  .thin {
+    font-size: 20px;
+    font-weight: 200;
+    text-transform: lowercase;
+    letter-spacing: 2px;
+  }
+
+  #header {
+    .description {
+      font-size: 20px;
+      font-weight: 900;
+      line-height: 18.84px;
```

Comment:
It is weird to use floats for such properties. We should use integers. Could you present us a difference in current look and the version with only ints used?

Reply:
I see no difference. Let's use integers